lib/cnet/udp/udp_input.c

```diff
@@ -86,20 +90,22 @@ udp_input_lookup(pktmbuf_t *m, struct pcb_hd *hd)
     else
         key.laddr.cin_addr.s_addr = uip->ip4.dst_addr;
 
-    md->faddr.cin_port = be16toh(uip->udp.src_port);
-    md->laddr.cin_port = be16toh(uip->udp.dst_port);
+    key.faddr.cin_port = uip->udp.src_port;
+    key.laddr.cin_port = uip->udp.dst_port;
 
     /* Create a 4x PCB lookup routine */
     pcb = cnet_pcb_lookup(hd, &key, BEST_MATCH);
     if (likely(pcb)) {
+#if 0        // Disable till cne_ipv4_udptcp_cksum_verify is fixed
         if ((pcb->opt_flag & UDP_CHKSUM_FLAG) && uip->udp.dgram_cksum) {
             if (is_pcb_dom_inet6(pcb))
                 csum = cne_ipv6_udptcp_cksum_verify(&uip->ip6, &uip->udp);
             else
                 csum = cne_ipv4_udptcp_cksum_verify(&uip->ip4, &uip->udp);
```

Comment:
So this checksum function call won't work because uip->udp is not a pointer into the mbuf - that is one issue with the checksum verify... I've fixed this up locally. will push something soon. 

Even with the fix though the checksum check is failing... 
Wireshark seems to be happy that the incoming packets from quicly have a good udp checksum 
![Screenshot 2023-09-28 at 10 13 37](https://github.com/CloudNativeDataPlane/cndp/assets/5012288/da378e11-acf9-4761-9eca-672042d3cebc)

Reply:
Ok, so I've figured it out... I fixed up the csum verification to be actually carried out on the mbuf itself... Will post a patch on top of this PR